src/test/java/net/datafaker/assertj/base/NameAssert.java

```diff
@@ -0,0 +1,39 @@
+package net.datafaker.assertj.base;
+
+import org.assertj.core.api.StringAssert;
+
+public class NameAssert extends StringAssert {
+
+    public NameAssert(String s) {
+        super(s);
+    }
+
+    public static NameAssert asserThat(String actual) {
+        return new NameAssert(actual);
+    }
+
+    public NameAssert isName() {
+        isNotNull();
+
+        // check condition
+        if (!actual.matches("([\\w']+\\.?( )?){2,4}")) {
```

Comment:
If the groups aren't reused then may as well make them non-capturing.

Reply:
You're probably right, but that's not really the purpose of this PR.